Services/UserService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Data;
+using Entities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Services
+{
+    public class UserService
+    {
+        public static Response AddUser(User user)
+        {
+            bool IsAdded = UserDataLayer.AddUser(user);
```

Comment:
where is validation?
what if i sent UserName ="" in request?

Reply:
Alright maam I'll take a look online and edit the file